build/tasks/RepoTasks.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
👍 repo tasks is ideal for "tools" that are only used in one place. Like SplitPackages. It doesn't really need to be a console tool.